pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -5473,8 +5512,23 @@ statfunc u32 cgroup_skb_submit_flow(struct __sk_buff *ctx,
             return 0;
     };
 
-    // Submit the flow base event so userland can derive the flow events.
-    cgroup_skb_submit(&events, ctx, neteventctx, event_type, size);
+    // Submit the flow event.
+    event_data_t *e = init_netflows_event_data();
+    if (unlikely(e == NULL))
+        return 0;
+    __builtin_memcpy(&e->context.task, &neteventctx->eventctx.task, sizeof(task_context_t));
+
+    save_to_submit_buf_kernel(&e->args_buf, (void *) &neteventctx->md.flow.proto, sizeof(u8), 0);
+    save_to_submit_buf_kernel(&e->args_buf, (void *) &neteventctx->md.flow.tuple, sizeof(tuple_t), 1);
+    save_to_submit_buf_kernel(&e->args_buf, (void *) &netflowvalue.bytes, sizeof(u64), 2);
+    save_to_submit_buf_kernel(&e->args_buf, (void *) &netflowvalue.packets, sizeof(u64), 3);
+    net_events_perf_submit(ctx, event_type, e);
+
+    // Reset stats after sample is submitted since we store diffs.
+    if (flow == flow_tcp_sample) {
+        reset_flow_stats(&netflowvalue);
+        bpf_map_update_elem(&netflowmap, &neteventctx->md.flow, &netflowvalue, BPF_NOEXIST);
```

Comment:
I do not think we there is the possibility to hit `BPF_NOEXISTS` with `flow_tcp_sample`, as we would return [here](https://github.com/castai/kvisor/pull/267/files#diff-64666057a3e407116fb615945a3f0e9b140b6a78a79cd355e972ba4f2d61f25cR5428). 

Anyway, I think it should be enough to call `reset_flow_stats` with the `netflowptr`, as this should update the memory region in the map (if I understand the code correct).